packages/mirinae/src/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs/blocks';
+import PCenteredLayoutHeader from '@/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.vue';
+import { getCenteredLayoutHeaderArgTypes } from "@/layouts/centered-layout/centered-layout-header/story-helper";
+
+
+<Meta title='Layouts/Centered Layout/Centered Layout Header' parameters={{
+    design: {
+        type: 'figma',
+        url: 'https://www.figma.com/file/IS6P8y1Wn2nfBC4jGlSiya/Components?node-id=10%3A228226&t=GVsFwo5txLv6VtoA-4'
+    }
+}} argTypes={getCenteredLayoutHeaderArgTypes()}
+/>
+
+export const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: {
+        PCenteredLayoutHeader
+    },
+    template: `
+    <div style="width: 360px">
+        <p-centered-layout-header :title="title"
+            :description="description"
+            :show-step="showStep"
+            :current-step="currentStep"
+            :total-steps="totalSteps"
+            :show-close-button="showCloseButton"
+        >
+        </p-centered-layout-header>
+    </div>
+    `,
+    setup() {
+        return {}
+    }
+});
+
+
+# Centered Layout Header
+
+<br/>
+<br/>
+
+## Basic
+
+<Canvas>
+    <Story name="Basic">
+        {{
+            components: {
+                PCenteredLayoutHeader
+            },
+            template: `
+            <div style="width: 360px">
```

Comment:
Suggestion: Personally, I would like it if the width was 100%. It looks like there is a fixed size.

![image](https://github.com/cloudforet-io/console/assets/50662170/668988a6-42e4-4c7e-b162-ec2266f0de58)

![image](https://github.com/cloudforet-io/console/assets/50662170/6e1b9693-75ef-4296-b536-151f2b2fa733)